src/css/styles.scss

```diff
@@ -10,16 +10,45 @@ $not-title-fonts: 'Montserrat';
 
```

Comment:
Add these variables in a separate file, variables.scss. Import that file where necessary